src/HDInsight/HDInsight/ManagementCommands/AddAzureHDInsightConfigValuesCommand.cs

```diff
@@ -22,9 +22,7 @@
 
 namespace Microsoft.Azure.Commands.HDInsight
 {
-    [GenericBreakingChange("Add-AzHDInsightConfigValues alias will be removed in an upcoming breaking change release", "2.0.0")]
```

Comment:
Why to remove breaking changes notice & cmdlets alias here?

Reply:
> Why to remove both breaking changes notice and cmdlets alias here? If its alias is to be removed, keep the notice, or keep the alias.

Hi @msJinLei , I found that the  breaking change notice said that the alias will be removed  at version 2.0.0. However now the stable version  is 2.0.1. So I think that maybe someone forgets to remove this alias and breaking change. So I removed it at this PR.

Reply:
> Why to remove both breaking changes notice and cmdlets alias here? If its alias is to be removed, keep the notice, or keep the alias.

Have added it back.